Appropriation Art Exhibit/exhibit.py

```diff
@@ -0,0 +1,697 @@
+__author__ = 'Harriet'
+
+import random
+import math
+import os
+
+from PIL import Image
+
+
+# The number of colour components, in this case, 3 (red, green and blue)
+NUMBER_OF_COLOR_COMPONENTS = 3
+
+RED = (255,0,0)
+GREEN = (0, 255, 0)
+BLUE = (0, 0, 255)
+BLACK = (0, 0, 0)
+
+
+def get_distance(first_point, second_point):
+    """Return distance between 2 coordinates as a float
+
+    Arguments:
+    first_point -- first coordinate tuple
+    second_point -- second coordinate tuple
+    """
+    x_distance = second_point[0] - first_point[0]
+    y_distance = second_point[1] - first_point[1]
+    distance = math.sqrt(x_distance**2.0 + y_distance**2.0)
+    return distance
+
+
+def show_gallery():
+    input_dir = "source-images"
+    output_dir = "output-images"
```

Comment:
Looking at the output images, the program certainly has great creative flare and produces very interesting and visually pleasing results.